src/screens/NoticeboardIndexScreen.tsx

```diff
@@ -0,0 +1,99 @@
+import { StackScreenProps } from '@react-navigation/stack';
+import React, { useCallback, useContext, useState } from 'react';
+import { useQuery } from 'react-apollo';
+import { ActivityIndicator, RefreshControl, SectionList } from 'react-native';
+
+import { EmptyMessage, LoadingContainer, SafeAreaViewFlex, SectionHeader } from '../components';
+import { colors, texts } from '../config';
+import { graphqlFetchPolicy, parseListItemsFromQuery } from '../helpers';
+import { useStaticContent } from '../hooks';
+import { useRenderItem } from '../hooks/listHooks';
+import { NetworkContext } from '../NetworkProvider';
+import { getQuery, QUERY_TYPES } from '../queries';
+
+import { ListHeaderComponent } from './NestedInfoScreen';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+/* eslint-disable complexity */
+export const NoticeboardIndexScreen = ({ navigation, route }: StackScreenProps<any>) => {
+  const { isConnected, isMainserverUp } = useContext(NetworkContext);
+  const fetchPolicy = graphqlFetchPolicy({ isConnected, isMainserverUp });
+  const [refreshing, setRefreshing] = useState(false);
+
+  const consentForDataProcessingText = route.params?.consentForDataProcessingText ?? '';
```

Comment:
what does this `consentForDataProcessingText` stand for and where will it be rendered?

Reply:
It is the text that the customer wants on the `Checkbox` we use for data processing permission. This will be displayed on the form.

`consentForDataProcessingText` is sent as the `titleDetail` prop in the `parseListItemsFromQuery` function and displayed on the Form page.

Reply:
is it possible to request the text directly on the form screen? why should it be handled on screens, where we do not need it?

Reply:
I don't think this is possible. The data here comes as param from the `noticeboardMainContent` `staticContent`. So we only have this data on this page. But if you want I can create a new `HtmlContent` just for this `consentForDataProcessingText`.